docs/src/manual/properties.rst

```diff
@@ -26,8 +26,8 @@ as possible, yet concise. Examples of such representations are for instance
 symmetry functions <Behler-Parrinello>`_. These representations are usually
 high-dimensional vectors, hard to visualize and interpret. For this reason, one
 usually applies a dimensionality reduction algorithm, such as `PCA`_, `sketch-map`_,
-*etc.*   The interpretation of the resulting  will differ depending on both the
-descriptor used to represent the structures or environments and the
+`PCovR`_, *etc.*   The interpretation of the resulting  will differ depending on
```

Comment:
'The interpretation of the resulting descriptor' in line 29 and remove ',' at the end of line 14